apps/plea/admin.py

```diff
@@ -371,8 +371,14 @@ def has_add_permission(self, request):
         return False
 
 
+class OffenceAdmin(admin.ModelAdmin):
```

Comment:
Do we need to add this? Given that this Django admin is exposed to admin staff it feels like we should be keeping things out of it unless absolutely necesarry for those admin staff.